Packs/Phishing/IncidentFields/incidentfield-Email_Internal_Message_ID.json

```diff
@@ -0,0 +1,36 @@
+{
+    "associatedToAll": false,
+    "associatedTypes": [
+        "Phishing",
+        "Phishing Alerts",
+        "FireEye EX Alert"
+    ],
+    "systemAssociatedTypes": [
+        "Phishing",
+        "Phishing Alerts",
+        "Email Communication"
```

Comment:
What is that field? Is email communication suppose to be here?

Reply:
Bad copy-paste, fixed